source/github-bugs.css

```diff
@@ -0,0 +1,30 @@
+/* Place Releases/Tags selector on the same line as `Draft new release` on releases page (https://github.com/sindresorhus/refined-github/commit/a34981cb3f0a1cf833f6e8f9f9591427d8f96473) */
+.subnav .d-flex.flex-md-row.flex-justify-between.flex-md-items-center {
+	float: left;
+}
+
+/* Align fork popup's header (https://github.com/sindresorhus/refined-github/pull/3352) */
+[aria-label^='Fork'] .Box-title {
+	position: static; /* The title wrongly overlaps the Close button, making it impossible to click */
+	min-height: 0;
+	margin: -5px 0;
+}
+
+/* Align author name on PR sticky header (https://github.com/sindresorhus/refined-github/pull/3444) */
+.sticky-content .author.css-truncate-target {
+	vertical-align: -4px;
+}
+
```

Comment:
```suggestion
```

> Before:
> ![image](https://user-images.githubusercontent.com/44045911/89752707-0d71bb00-db08-11ea-9f50-e625c956bf11.png)

I'm not seeing this issue anywhere, with or without RG

Regular: <img width="211" alt="Screen Shot 2020-08-10 at 10 12 02" src="https://user-images.githubusercontent.com/1402241/89767597-2d967f80-daf2-11ea-84ee-8cabb555eee6.png">


Stickied: <img width="601" alt="Screen Shot 2020-08-10 at 10 22 53" src="https://user-images.githubusercontent.com/1402241/89768530-9d593a00-daf3-11ea-816f-45793b38c3bb.png">

Reply:
This happens on my Edge and Chrome, but not on Firefox. @yakov116 do you have this problem?

Reply:
Maybe how Chromium handles `vertical-align: top` is different from Firefox. I added `vertical-align: -4px` to Firefox and the text's position doesn't change.

Reply:
You likely have some conflicting styles. I'm not seeing that in any browser and I don't expect it to last. It should be dropped from this PR

Reply:
> You likely have some conflicting styles

How is that possible? I tested it in a newly installed Chrome

Reply:
🤷‍♂️ 

<img width="500" alt="Screen Shot 2020-08-10 at 16 53 43" src="https://user-images.githubusercontent.com/1402241/89802946-16738400-db2a-11ea-8a58-34e02fcb5420.png">

Reply:
I think I figured out: latest stable Chromium doesn't have this issue, but Chromium Dev does (which affects both Chrome and Edge)